addons/website/static/src/components/views/page_views_mixin.js

```diff
@@ -32,6 +33,8 @@ export const PageControllerMixin = (component) => class extends component {
             await this.website.fetchWebsites();
             this.websiteSelection.push(...this.website.websites);
             this.state.activeWebsite = this.website.currentWebsite || this.website.websites[0];
+            const pageIds = await this._getWebsitePageIds(this.state.activeWebsite.id);
+            this.env.searchModel.updateWebsitePageIds(pageIds.toString());
```

Comment:
Particularity: I have to pass the array as string here, and `.split(',')` it in the searchModel when sending it to the server, otherwise I have an infinite loop. I'll investigate about that later.